cluster.yml

```diff
@@ -206,18 +206,6 @@
 # ENDBLOCK # Create Cluster Service Infrastructure
 # STARTBLOCK # Prepare TLS
 
-- name: Fetch CA certificates
```

Comment:
I'm not very familiar with the inner works of this, but IIUC the certificates fetched here are used later for the building of the keystores and truststores that are distributed to the hosts, right?

By moving these tasks from here into the kerberos-related ones, does that mean that this will break deployments that have a `ca_server` but _not_ a `krb5_server`? In that case, how will the certificates from the CA server be fetched to be used in the next phases of the playbook?